gpu_multi_process_run.sh

```diff
@@ -32,10 +32,15 @@ set_nccl_gpudirect_tcpx_specific_configuration() {
     export TF_CPP_MAX_LOG_LEVEL=100
     export TF_CPP_VMODULE=profile_guided_latency_estimator=10
     export XLA_PYTHON_CLIENT_MEM_FRACTION=0.85
+    shopt -s globstar nullglob
+    IFS=:$IFS
+    set -- /usr/local/cuda-12.*/compat
```

Comment:
Should this be `cuda-*` instead of tying to `12`?

Reply:
I could do that. It should still work with cuda-12.* as well